scout/config.py

```diff
@@ -58,14 +104,37 @@ def _validate(self, input_data: dict, schema_data: dict = None):
             schema_data = self.schema_data
         validate(input_data, schema_data)
 
+    def _resolve_filepath_args(self):
```

Comment:
Required so that --ecm_directory can be defined as an absolute path or relative, and either via the CLI or the yml file.